marlowe-website/src/index.js

```diff
@@ -33,7 +33,27 @@ function initializeHeaderComponent() {
     }
   };
 }
+
+function fixCrossLinks() {
+  const env = window.location.hostname.split('.')[0];
+
+  if (env == 'localhost' || env == 'marlowe-finance') {
+    return;
+  }
+
+  const urls = {
+    'run': 'https://' + env + '.marlowe-dash.iohkdev.io',
+    'play': 'https://' + env + '.marlowe.iohkdev.io'
+  };
+
+  const crossLinks = document.getElementsByClassName("cross-link");
```

Comment:
Small thing, but I prefer not to use class names as ways of identifying elements (unless you mean to get all the elements that have a certain style). This feels nicer to me:

```suggestion
  const crossLinks = document.querySelectorAll("[data-marlowe-component]");
```

Reply:
Fixed.